client/src/locales/gb.json

```
@@ -445,73 +445,623 @@
+  "passwordPlaceholder": "••••••••••",
+  "monitorDeletedSuccessfully": "Monitor deleted successfully.",
+  "monitorCreatedSuccessfully": "Monitor created successfully!",
+  "failedToCreateMonitor": "Failed to create monitor.",
+  "maintenanceWindowDeletedSuccessfully": "Maintenance window deleted successfully.",
+  "failedToDeleteMaintenanceWindow": "Failed to delete maintenance window.",
+  "failedToPauseMaintenanceWindow": "Failed to pause maintenance window.",
+  "failedToFetchData": "Failed to fetch data",
+  "failedToCreateStatusPage": "Failed to create status page",
+  "failedToDeleteStatusPage": "Failed to delete status page",
+  "monitorState": {
+    "paused": "paused",
```

Biome (1.9.4)

[error] The key monitorState was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)